include/spdlog/sinks/systemd_sink.h

```diff
@@ -8,6 +8,7 @@
 #include "spdlog/details/synchronous_factory.h"
 
 #include <array>
+#define SD_JOURNAL_SUPPRESS_LOCATION
```

Comment:
#ifndef SD_JOURNAL_SUPPRESS_LOCATION
#define SD_JOURNAL_SUPPRESS_LOCATION
#endif

Reply:
otherwise we can get ugly warnings..

Reply:
done